### PR TITLE
pr demo

### PR DESCRIPTION
**Pull request by gmkhach**
This is a demo of how pull requests work
 
Usually the requester will list all of the changes that the pr is introducing to the base branch. 
The changes will be listed in a clean, easy to read, format.

The next section will have clear instructions on how to test the app for proper functionality of the new features, or bug fixes, or whatever else the pr is addressing. You can also include snapshots and .gifs to assist your instructions.

### CODE REVIEW

File: setup/db.sql [hunk 1]
```diff
@@ -1,7 +1,8 @@
 -- create schema
 drop schema if exists library cascade;
-create schema if not exists library;
+create schema library;
```
Comment (lilitkhamalyan) [inline setup/db.sql:4]: test issue

Comment (gmkhach) [reply setup/db.sql:4]: simulate addressing the test issue with a new commit that solves it

Comment (lilitkhamalyan) [reply setup/db.sql:4]: LGTM

**Comment by lilitkhamalyan** [comment]
LGTM

**Pull request merged**